### PR TITLE
fix(dropdown): update state of dropdown option

### PR DESCRIPTION
**Pull request by varnikajain15**
### What does this implement/fix? Explain your changes.
Update active state of selected option in dropdown 
...

### Does this close any currently open issues?

...

### Any other comments?

...

### Dependent PRs/Commits

...

### Describe breaking changes, if any.

...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

File: core/components/atoms/dropdown/option/IconOption.tsx [hunk 1]
```diff
@@ -25,9 +24,7 @@ const IconOption = (props: OptionTypeProps) => {
       data-disabled={disabled}
     >
       {/* eslint-enable  */}
-      {icon && (
-        <Icon className="Option-icon mr-4" data-test={`${dataTest}--Icon`} name={icon} appearance={appearance} />
-      )}
+      {icon && <Icon className="Option-icon mr-4" data-test={`${dataTest}--Icon`} name={icon} />}
```
Comment (Rashi-Gupta-2000) [inline core/components/atoms/dropdown/option/IconOption.tsx:27]: Please do not remove appearance but just override the active for selected in CSS.

File: css/src/components/dropdown.css [hunk 2]
```diff
@@ -115,20 +116,27 @@
 
 .Option--selected {
   background-color: var(--primary-lightest);
+  color: var(--primary-dark);
```
Comment (Rashi-Gupta-2000) [inline css/src/components/dropdown.css]: Please remove these from CSS.

**Review by Rashi-Gupta-2000 — DISMISSED** [review]

File: core/components/atoms/dropdown/option/index.tsx [hunk 3]
```diff
@@ -165,12 +170,18 @@ const Option = (props: OptionProps) => {
 
   const renderSubInfo = (subInfo: string | MetaListProps) => {
     const labelAppearance = disabled ? 'disabled' : selected && !menu ? 'white' : 'subtle';
-    const color = disabled ? 'inverse-lightest' : selected && !menu ? 'primary-dark' : 'inverse';
+    const color = disabled ? 'inverse-lightest' : selected && !menu ? 'primary-dark' : 'inverse-lighter';
```
Comment (Rashi-Gupta-2000) [inline core/components/atoms/dropdown/option/index.tsx]: There are 2 same variable names please update them to be specific.

File: core/components/atoms/dropdown/option/index.tsx [hunk 4]
```diff
@@ -98,13 +98,18 @@ const Option = (props: OptionProps) => {
...
+  const type = checkboxes ? 'WITH_CHECKBOX' : optionType;
+  const component = OptionTypeMapping[type];
 
   const OptionClassName = classNames({
     ['Option']: true,
     ['Option--active']: active,
     ['Option--selected']: selected && !menu,
     ['Option--disabled']: disabled,
     ['OptionWrapper']: true,
+    [`color-${color}`]: true,
```
Comment (Rashi-Gupta-2000) [inline core/components/atoms/dropdown/option/index.tsx:112]: Please write test case for the added class.

**Review by Rashi-Gupta-2000 — APPROVED** [review]

**Review by anuradha9712 — APPROVED** [review]

**Pull request merged**